core/app/models/spree/order.rb

```diff
@@ -26,7 +26,14 @@ class Order < Spree::Base
     include Spree::Order::Checkout
     include Spree::Order::Payments
 
-    class InsufficientStock < StandardError; end
+    class InsufficientStock < StandardError
+      attr_reader :items
+
+      def initialize(message = nil, items: [])
```

Comment:
Should the default value of `items:` be an array or a hash?

Reply:
True! it returns a hash of items